include/exadg/postprocessor/time_control.h

```diff
@@ -0,0 +1,99 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_COMPRESSIBLE_NAVIER_STOKES_POSTPROCESSOR_TIME_CONTROL_H_
+#define INCLUDE_COMPRESSIBLE_NAVIER_STOKES_POSTPROCESSOR_TIME_CONTROL_H_
+
+/**
+ *This class provides information at which time steps a certain action should be triggered. It is
+ *typically used in postprocessing routines that shall not be invoked after every time step. The
+ *user can specify an evaluation interval in terms of number of time steps or an amount of time.
+ */
+
+// deal.ii
+#include <deal.II/base/conditional_ostream.h>
+
+// ExaDG
+#include <exadg/utilities/numbers.h>
+
+namespace ExaDG
+{
+struct TimeControlData
+{
+  TimeControlData();
```

Comment:
where are the default values? trigger_interval and trigger_every_time_steps should be set to invalid values by default in my opinion.

Reply:
They are set in the constructor in the cpp file.